channels/wager

```diff
@@ -2,8 +2,10 @@
 
 You can decide to either:
 
-1. Kill someone, and sell your soul in the process
+1. Kill someone, but sell your soul in the process
 
-2. Check someone's role, but the Devil will also see the result.
+2. Protect someone, but sell your soul in the process
```

Comment:
maybe include a "(including yourself)" since it you can also protect yourself and it kinda sounds like you cant.